python/pyarrow/tests/parquet/test_data_types.py

```diff
@@ -510,3 +510,16 @@ def test_large_binary_overflow():
                 pa.ArrowInvalid,
                 match="Parquet cannot store strings with size 2GB or more"):
             _write_table(table, writer, use_dictionary=use_dictionary)
+
+
+@pytest.mark.parametrize("storage_type", (
+    pa.utf8(), pa.large_utf8(), pa.string(), pa.large_string()))
```

Comment:
```suggestion
    pa.string(), pa.large_string()))
```

No need to parametrize with both ut8 and string, as they are just aliases